content/04.study.md

```diff
@@ -402,7 +402,7 @@ We are excited to see the creative applications of deep learning to single-cell
 Metagenomics, which refers to the study of genetic material -- 16S rRNA or whole-genome shotgun DNA -- from microbial communities, has revolutionized the study of micro-scale ecosystems within and around us.
 In recent years, machine learning has proved to be a powerful tool for metagenomic analysis.
 16S rRNA has long been used to deconvolve mixtures of microbial genomes, yet this ignores more than 99% of the genomic content.
-Subsequent tools aimed to classify 300 bp-3000 bp reads from complex mixtures of microbial genomes based on tetranucleotide frequencies, which differ across organisms [@tag:Karlin], using supervised [@tag:McHardy; @tag:nbc] or unsupervised methods [@tag:Abe].
+Subsequent tools aimed to classify 300 bp--3000 bp reads from complex mixtures of microbial genomes based on tetranucleotide frequencies, which differ across organisms [@tag:Karlin], using supervised [@tag:McHardy; @tag:nbc] or unsupervised methods [@tag:Abe].
```

Comment:
Should this be simplified to `300--3000 bp`?

Reply:
OK. Searched everything for `[A-Za-z]--?[0-9]` and that was the only relevant match.